CHANGELOG.md

```diff
@@ -14,6 +15,106 @@
+- The `NamedAddress` value for `Gateway`'s `spec.addresses[].type` field has
+  been deprecated, and support for domain-prefixed values (like
+  `example.com/NamedAddress`) has been added instead to better represent the
+  custom nature of this support.
+  [#1178](https://github.com/kubernetes-sigs/gateway-api/pull/1178)
+- `GatewayTLSConfig`'s `CertificateRefs` field is now a slice of pointers to
+  structs instead of the structs directly.
+  [#1176](https://github.com/kubernetes-sigs/gateway-api/pull/1176)
+- Webhook validation now ensures that a path match exists when required by path
+  modifier in filter.
+  [#1171](https://github.com/kubernetes-sigs/gateway-api/pull/1171)
+- Implementations are now expected to use `404` instead of `503` responses when
+  the data-plane has no matching route.
+  [#1151](https://github.com/kubernetes-sigs/gateway-api/pull/1151)
+- The `UnsupportedExtension` named `ListenerConditionReason` has been removed.
+  [#1146](https://github.com/kubernetes-sigs/gateway-api/pull/1146)
+- The `RouteConflict` named `ListenerConditionReason` has been removed removed.
+  [#1145](https://github.com/kubernetes-sigs/gateway-api/pull/1145)
+- The `URLRewrite` filter `PreciseHostname` has replaced `Hostname`
+  [#1124](https://github.com/kubernetes-sigs/gateway-api/pull/1124)
+- Webhook validation was added to ensure that only type-appropriate fields are
+  set in `HTTPPAthModifier`.
+  [#1124](https://github.com/kubernetes-sigs/gateway-api/pull/1124)
+- `HTTPPathModifer` field `Absolute` renamed to `ReplaceFullPath`
+  [#1124](https://github.com/kubernetes-sigs/gateway-api/pull/1124)
+- `HTTPPathModifier` API replaces `Substitution` with new type-specific
+  `ReplaceFullPath` and `ReplacePrefixMatch` fields
+  [#1124](https://github.com/kubernetes-sigs/gateway-api/pull/1124)
+- Types `ConditionRouteAccepted` and `ConditionRouteResolvedRefs` are now
+  deprecated in favor of `RouteConditionAccepted` & `RouteConditionResolvedRefs`
+  [#1114](https://github.com/kubernetes-sigs/gateway-api/pull/1114)
+- The Gateway API webhook is now deployed in a `gateway-system` namespace
+  instead of `gateway-api`.
+  [#1051](https://github.com/kubernetes-sigs/gateway-api/pull/1051)
+- the `ParentRef` type was renamed to `ParentReference`
+  [#982](https://github.com/kubernetes-sigs/gateway-api/pull/982)
+- Gateway API now includes "Experimental" release channel
+  [#945](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+- Gateway API CRDs now include `gateway.networking.k8s.io/bundle-version` and
+  `gateway.networking.k8s.io/channel` annotations
+  [#945](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+- URL Rewrites and Path redirects have been added as new "Experimental" features
+  [#945](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+- The v1alpha1 API version has been deprecated.
+  [#906](https://github.com/kubernetes-sigs/gateway-api/issues/906)
```

Comment:
This is actually a good catch, I think we'd previously discussed removing it in v0.5.0, any thoughts?

cc @youngnick

Reply:
Covered by https://github.com/kubernetes-sigs/gateway-api/pull/1197

Reply:
Yes, agreed that v1alpha1 should be gone.

Reply:
Updated in for the removal that occurred in https://github.com/kubernetes-sigs/gateway-api/pull/1197